src/openrct2/network/NetworkBase.cpp

```diff
@@ -1270,14 +1270,14 @@ void NetworkBase::Server_Send_OBJECTS_LIST(
     }
 }
 
-void NetworkBase::Server_Send_SCRIPTS(NetworkConnection& connection) const
+void NetworkBase::Server_Send_SCRIPTS(NetworkConnection& connection)
```

Comment:
(On https://github.com/OpenRCT2/OpenRCT2/pull/15232/commits/b3bb2f5d80765bf383fd60d5bd5b2c79fd0dcbba)
Why was the constness droppped?

Maybe because it isn't called the `const` `GetContext()`?

Reply:
Yeah I had issues getting this to const so I had to drop it for now, not happy about it but the other way would be much worse.